lib/CXGN/Project.pm

```diff
@@ -3335,6 +3356,12 @@ sub create_tissue_samples {
                         type_id => $tissue_sample_cvterm,
                     });
 
+                    if ($tissue_sample_owner) {
+                        my $q = "INSERT INTO phenome.stock_owner(stock_id, sp_person_id) values(?,?)";
```

Comment:
should use the stock object for the insert CXGN::Stock 
or use CXGN::Phenome::Schema::StockOwner